WalletWasabi/Extensions/ProcessExtensions.cs

```diff
@@ -70,5 +71,14 @@ private static void Process_Exited(object sender, EventArgs e)
 				Logger.LogError(ex, nameof(ProcessExtensions));
 			}
 		}
+
+		public static TaskAwaiter<int> GetAwaiter(this Process process)
```

Comment:
Remove this and use `WaitForExitAsync` instead.

Reply:
I saw the `WaitForExitAsync` but it does more things, I wanted to be sure I don't introduce breaking changes. I mean, using `WaitForExitAsync` is a bigger change.

I will change it but I am not sure, it is hard to test i think.

Reply:
Done.